Sources/HaCWebsiteLib/Utils/DatabaseUtils.swift

```diff
@@ -10,6 +10,10 @@ struct DatabaseURLComponents {
   let database: String
 }
 
+private let preparations = [
```

Comment:
@Pinpickle what is this here for?

Reply:
Remnants of a prior implementation, I'll get rid of this